unstructured/ingest/main.py

```diff
@@ -221,6 +223,13 @@ def main(
             ),
         )
     elif github_url:
+        if not download_dir:
+            download_dir = (
+                cache_path
+                / hashlib.md5(
+                    f"{github_url}_{github_branch}_{github_file_glob}".encode("utf-8"),
```

Comment:
actually the glob shouldn't matter. if downloading "*html,*txt" in one run, in "*html" in another, i'd expect it to not redownload the html for the same github url and branch.

Reply:
Yeah probably that makes more sense, I just went fast over that connector because I was not super aware of it, but I'll remove that part now :)

P.S. Can you send me your no-reply email to add you as a co-author? or are you willing to add a suggestion so that I commit it directly through GitHub?

Reply:
> P.S. Can you send me your no-reply email to add you as a co-author? or are you willing to add a suggestion so that I commit it directly through GitHub?

Thank you for asking -- I'm ok just being a reviewer on this one 😊